.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,86 +1,75 @@
+  - Update .po files using [PoExtractor](https://github.com/lukaskabrt/PoExtractor) to refresh [Crowdin](https://crowdin.com/project/orchard-core).
+  - Publish the new version on NuGet.
+  - Update the `OrchardCore.Translations.All` package reference in the main repository's `./Dependencies.Packages.props` file.
 
-Make sure everything works all right.
+## Creating the New Release
 
-- [ ] Make sure that [`OrchardCore.Samples` works](https://github.com/OrchardCMS/OrchardCore.Samples).
-- [ ] Test the [guides](https://docs.orchardcore.net/en/latest/guides/) with the NuGet packages from the Cloudsmith feed (branches under `release/` are automatically published too). Test at least the following guides:
-  - [Creating a modular ASP.NET Core application](https://docs.orchardcore.net/en/latest/guides/create-modular-application-mvc/)
-  - [Creating an Orchard Core CMS website](https://docs.orchardcore.net/en/latest/guides/create-cms-application/)
-  - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/)
-- [ ] Re-certify Orchard Core for the latest major version of Red Hat Enterprise Linux if a new version has been released (e.g., v10 after v9). Refer to Orchard's [Red Hat Ecosystem Catalog profile](https://catalog.redhat.com/software/applications/detail/223797) for the previously certified version, the [Red Hat Customer Portal](https://access.redhat.com/articles/3078) for the latest version, and [our certification guide](https://docs.orchardcore.net/en/latest/topics/red-hat-ecosystem-catalog-certification/) for the certification steps.
-
-### Prepare and publish Orchard Core Translations
-
-Update everything in the [Translations project](https://github.com/OrchardCMS/OrchardCore.Translations). Only do this once all the code changes are done since localized strings can change until then.
-
-- [ ] Update .po files with [PoExtractor](https://github.com/lukaskabrt/PoExtractor). This will also update [Crowdin](https://crowdin.com/project/orchard-core).
-- [ ] Publish the new version on NuGet.
-- [ ] Update the `OrchardCore.Translations.All` package reference in the main repo's _./Dependencies.Packages.props_ file to refer to the new NuGet package.
-
-### Publish the release
+1. Navigate to the [GitHub Releases page](https://github.com/OrchardCMS/OrchardCore/releases/new).
+2. In the "**Choose a tag**" menu, enter the new version number (e.g., `v3.0.0`) and select "**+ Create tag: v... on publish**."
+3. Change the target branch from `main` to your release branch (e.g., `release/3.0`).
+4. Enter the version number in the Title field (e.g., `3.0.0`).
+5. Click **Generate release notes** to automatically create release notes.
+6. Ensure the "Set as the latest release" checkbox is checked, then click **Publish release**.
 
-Do the harder parts of making the release public. This should come after everything above is done.
+## Validation
```

Comment:
This should happen before the release, with the preview feed, so move it before Creating the New Release. Same in the other template.